cmd/cloud/installation.go

```diff
@@ -164,6 +172,56 @@ var installationDeleteCmd = &cobra.Command{
 	},
 }
 
+var installationHibernateCmd = &cobra.Command{
+	Use:   "hibernate",
+	Short: "Put an installation into hibernation.",
+	RunE: func(command *cobra.Command, args []string) error {
+		command.SilenceUsage = true
+
+		serverAddress, _ := command.Flags().GetString("server")
+		client := model.NewClient(serverAddress)
+
+		installationID, _ := command.Flags().GetString("installation")
+
+		installation, err := client.HibernateInstallation(installationID)
+		if err != nil {
+			return errors.Wrap(err, "failed to put installation into hibernation")
+		}
+
+		err = printJSON(installation)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	},
+}
+
+var installationWakeupCmd = &cobra.Command{
+	Use:   "wake-up",
+	Short: "Wake an installation from hibernation.",
+	RunE: func(command *cobra.Command, args []string) error {
+		command.SilenceUsage = true
+
+		serverAddress, _ := command.Flags().GetString("server")
+		client := model.NewClient(serverAddress)
+
+		installationID, _ := command.Flags().GetString("installation")
+
+		installation, err := client.WakupInstallation(installationID)
```

Comment:
`WakupInstallation` -- pretty sure you want `WakeupInstallation`

Reply:
But everyone loves `Wakup`!